src/applications/disability-benefits/526EZ/tests/helpers.unit.spec.jsx

```diff
@@ -2,52 +2,41 @@ import sinon from 'sinon';
 import { expect } from 'chai';
 import {
   flatten,
-  isPrefillDataComplete,
+  transformDisabilities,
   prefillTransformer,
   get4142Selection,
   queryForFacilities
 } from '../helpers.jsx';
 import initialData from './schema/initialData.js';
 
-delete initialData.prefilled;
-const treatments = [
-  {
-    treatment: {
-      treatmentCenterName: 'local VA center'
-    }
-  }
-];
-initialData.disabilities[0].treatments = treatments;
-const flattened = flatten(initialData);
-const completeData = initialData;
-const incompleteData = {};
-const { formData: transformedPrefill } = prefillTransformer([], initialData, {}, { prefilStatus: 'success' });
-const { formData: incompletePrefill } = prefillTransformer([], {}, {}, { prefilStatus: 'success' });
-
 describe('526 helpers', () => {
+  const treatments = [
+    {
+      treatment: {
+        treatmentCenterName: 'local VA center'
+      }
+    }
+  ];
+  const prefilledData = Object.assign({}, initialData);
+  prefilledData.disabilities[0].treatments = treatments;
+  const flattened = flatten(prefilledData);
+  const { formData: transformedPrefill } = prefillTransformer([], prefilledData, {}, { prefilStatus: 'success' });
```

Comment:
`prefilStatus` -> `prefillStatus`

I see it's a typo from the old version too, but probably should rename.

Reply:
Thanks @cvalarida, this isn't used anymore actually, so I will remove it.